contracts/ZoraModuleManager.sol

```diff
@@ -3,6 +3,13 @@ pragma solidity 0.8.10;
 
 import {ZoraProtocolFeeSettings} from "./auxiliary/ZoraProtocolFeeSettings/ZoraProtocolFeeSettings.sol";
 
+struct ModuleApprovalSig {
+    uint8 v; // The 129th byte and chain ID of the signature
+    bytes32 r; // The first 64 bytes of the signature
+    bytes32 s; // Bytes 64-128 of the signature
```

Comment:
Should these comments instead say 65th byte, first 32 bytes, and bytes 32-64, respectively? Or am I missing another part of the signature?

Reply:
Lmao I honestly didn't even expect to have review permission on this.

Reply:
I think it's correct – from the [EIP-712 spec](https://eips.ethereum.org/EIPS/eip-712):

> As in eth_sign it is a hex encoded 129 byte array starting with 0x. It encodes the r, s and v parameters from appendix F of the [yellow paper](https://ethereum.github.io/yellowpaper/paper.pdf) in big-endian format. Bytes 0…64 contain the r parameter, bytes 64…128 the s parameter and the last byte the v parameter. Note that the v parameter includes the chain id as specified in [EIP-155](https://eips.ethereum.org/EIPS/eip-155)[](https://eips.ethereum.org/EIPS/eip-712#example).

Reply:
Ohh yea the ERC is very unclear with the terminology there. Seems it confuses bytes with hex characters in that paragraph (the appendix it references also says there are 65 bytes in a sig). Thanks for linking!

Reply:
Lol it also said to expect a 129 byte array as a signature but shows a 65 byte array in the example just below?? I’m gonna try and open a PR over there